src/Control/Effect/Catch.hs

```diff
@@ -33,11 +33,7 @@ instance Effect (Catch e) where
 
 -- | Run a computation which can throw errors with a handler to run on error.
 --
--- Errors thrown by the handler will escape up to the nearest enclosing 'catchError' (if any).
--- Note that this effect does /not/ handle errors thrown from impure contexts such as IO,
--- nor will it handle exceptions thrown from pure code. If you need to handle IO-based errors,
--- consider if 'Control.Effect.Resource' fits your use case; if not, use 'Control.Monad.IO.Class.liftIO' with
--- 'Control.Exception.try' or use 'Control.Exception.catch' from outside the effect invocation.
+-- Errors thrown by the handler will escape up to the nearest enclosing 'catchError' (if any). Note that this effect does /not/ handle errors thrown from impure contexts such as IO, nor will it handle exceptions thrown from pure code. If you need to handle IO-based errors, consider if @fused-effects-exceptions@ fits your use case; if not, use 'Control.Monad.IO.Class.liftIO' with 'Control.Exception.try' or use 'Control.Exception.catch' from outside the effect invocation.
```

Comment:
1. Soft-wrapped.

2. Replaced a reference to `Control.Effect.Resource` with a reference to `fused-effects-exceptions`.